utils/check_repo.py

```diff
@@ -102,6 +102,9 @@
 # should **not** be the rule.
 IGNORE_NON_AUTO_CONFIGURED = PRIVATE_MODELS.copy() + [
     # models to ignore for model xxx mapping
+    "ViltForMaskedLM",
```

Comment:
`AutoModelForMaskeLM` should correctly return this, no?

Reply:
I asked @Narsil about this, but the `AutoModelForMaskedLM` doesn't currently accept models that take several modalities as input.

Reply:
The pipeline `fill-mask` doesn't work with multi modalities. And it assumes every `AutoModelForMaskedLM` is for filling text only masks.
As I mentionned too:

- Either we don't make it `AutoMaskedLM`
- Or we make it `AutoMaskedLM` but we need an escape hatch of some kind so that the pipeline can know it's not supposed to work.
- `AutoModel` should work nonetheless (I assume this discards that)